Tests/LibGfx/TestImageDecoder.cpp

```diff
@@ -768,6 +769,18 @@ TEST_CASE(test_png)
     TRY_OR_FAIL(expect_single_frame(*plugin_decoder));
 }
 
+TEST_CASE(test_exif)
+{
+    auto file = TRY_OR_FAIL(Core::MappedFile::map(TEST_INPUT("png/exif.png"sv)));
```

Comment:
You might've forgotten to `git add` this file. (Please mention how you created it somewhere.)

Reply:
Oops, sorry about that!